remake.yml

```diff
@@ -8,22 +8,29 @@ packages:
   - cowplot
   - scipiper
   - purrrlyr
+  - leaflet
+  - leafpop
+  - htmlwidgets
 
 sources:
   - 1_fetch/src/find_oldest_sites.R
   - 1_fetch/src/get_site_data.R
   - 3_visualize/src/map_sites.R
   - 123_state_tasks.R
+  - 3_visualize/src/plot_data_coverage.R
+  - 3_visualize/src/map_timeseries.R
 
 targets:
   main:
     depends:
       - 3_visualize/out/site_map.png
-      - state_tasks
+      - state_combiners
```

Comment:
You don't actually need state_combiners here since it's a dependency of the map and plot.